package.json

```diff
@@ -74,7 +74,7 @@
     "address": "^1.2.2",
     "bcryptjs": "^2.4.3",
     "better-sqlite3": "^8.3.0",
-    "cachified": "^3.4.1",
+    "cachified": "^3.5.3",
```

Comment:
I've added `mergeReporters` and the `"done"` event in `v3.5` _(and then had a build issue for the next 3 releases 🙈)_